docs/fundamentals/tutorials/new_executor.md

```diff
@@ -0,0 +1,370 @@
+# Executor: From Zero To Hub
+
+> A step by step walkthrough on how to create your Executors or use existing ones.
+
+[comment]: <> (TODO add link to the chatbot tutorial when it's moved here )
+
+[comment]: <> (Last time we talked about how to create the [hello world chatbot]&#40;https://jina.ai/tutorial.html&#41;, but we didn't go much into Executors' details. Let's take a look at them now.)
+
+We will create a simple logging Executor. It will log the Documents' information as they reach it, and save these to a file. We will also see how to push our Executor to Jina Hub to use it later.
+
+## Set-up & overview
+
+We recommend creating a [new python virtual environment](https://docs.python.org/3/tutorial/venv.html) to have a clean installation of Jina and prevent dependency clashing.
+
+We can start by installing Jina:
+
+ ```bash
+pip install jina
+ ```
+
+For more information on installing Jina, refer to this [page](https://github.com/jina-ai/jina#install).
```

Comment:
````suggestion
```{admonition} See Also
:class: seealso
For more information on installing Jina, refer to the {ref}`installation page <installation>`
```
````
and in the docs installation section (get-started/install.md) do this change:
```
(installation)=
# Install
```